lib/ontologies_linked_data/models/mod/semantic_artefact_distribution.rb

```diff
@@ -102,9 +113,15 @@ def bring(*attributes)
                     when :ontology_submission
                         @submission.bring(*mapped_attr)
                         self.send("#{attr}=", @submission.send(mapped_attr)) if @submission.respond_to?(mapped_attr)
-                    when :metrics
-                        next
-                        # TO-DO
+                    when :metric
```

Comment:
duplicate should be extract as module/concern, you can put it here https://github.com/ontoportal-lirmm/ontologies_linked_data/tree/master/lib/ontologies_linked_data/concerns and see the existent files as example.

Reply:
Summary: extract all duplicates code in one place, and re-use it.